### DISCUSSION TITLE
Train the model

### DISCUSSION THREAD

**Discussion by deepaks11** [ultralytics/ultralytics#4286] [Q&A]
1. Can I train the facial emotion dataset using YOLOv8?
2. Can I train a model to recognize different types of faces along with their names and identify the faces in real-time?

**Comment by suhailshaji** [comment]
@deepaks11  Yes, all you need is a dataset that can be used to train YOLOV8. You may obtain similar datasets or convert common coco formats to YOLOV8 formats.
check the repo of a sample face dataset : [https://www.kaggle.com/datasets/lylmsc/wider-face-for-yolo-training](https://www.kaggle.com/datasets/lylmsc/wider-face-for-yolo-training)

hope helps!
